src/main/java/se/audit/aop/AuditAspect.java

```diff
@@ -69,7 +70,11 @@ private boolean isLoggable(Parameter parameter) {
     }
 
     private String truncateString(String longString) {
-        String limitedString = longString.substring(0, Math.min(longString.length(), auditProperties.getMaxOutputLength()));
+        String limitedString = null;
+        try {
+            limitedString = longString.substring(0, Math.min(longString.length(), auditProperties.getMaxOutputLength()));
+        } catch (Exception e) {
+        }
         return limitedString.equals(longString) ? limitedString : limitedString + TRUNCATED_SUFFIX;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "limitedString" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2259)